### ISSUE TITLE
Dedicated Alda Slack group for chat / support

### ISSUE THREAD

**Issue by daveyarwood** [alda-lang/alda#288]
I'm over a year late on this, but the more I think about it, the more I like @crisptrutski / @MadcapJake 's idea in #86.

> Still wondering whether gitter or creating a dedicated slack community (see http://rauchg.com/slackin/) wouldn't be more approachable, flexible and future-proof..

> A gitter would be nice but I think maybe a dedicated slack makes more sense than even the current Clojurian channel:
> * (re: Gitter) Slack is a little more non-tech crowd friendly—no need for a github account.
> * (re: current channel) In a perfect world all musicians would be clojurians, but I'm afraid that's not the case in ours :stuck_out_tongue_winking_eye: . 
> * A dedicated slack means that Alda can be the focus, front-and-center on the first channel the users enter.  Slack may be more friendly to non-tech users than Gitter but it certainly isn't perfect.  There's a couple of steps left after registering and logging in before you're in the > `#Alda` channel. 
> * I don't know the details of Clojurians' Slack registration, but for free acounts, there is a limited number of messages. Perhaps the Clojurian users wouldn't want non-clojure users contributing against that limit.

I'm totally on-board with this idea. I think if we can make it super easy for newcomers to hop right into an Alda chatroom, it could go a long way toward helping us foster a sense of community amongst Alda users.

I've set up a free Slack group (alda-lang.slack.com), currently invite-only, but when I have some time to fiddle with the settings, I'd like to make it open to the public and get [slackin](http://rauchg.com/slackin/) set up so that it's super easy to join from both our GitHub repo and the [alda website](http://alda.io) (under construction).

If anyone's feeling industrious, a PR for either of those things (slackin integrated into both the Alda website and this repo's README) would be fantastic, otherwise I can get crackin' on this whenever I have some free time.

P.S. The Alda website GitHub repo is [here](https://github.com/alda-lang/alda-lang.github.io)

**Comment by nkonk** [comment]
#289 

I created a Slackin instance on Cloud9.(Intended to do it on Heroku) but for some reason the dashboard wasn't loading here)

For now, it runs on a cloud9 public instance which means it will go down when the instance is put into hibernation(which cloud9 does for free accounts).

For now it just connects to a dummy slack organisation set up by me(test-slackin-alda).

I also have the iframe code for putting on to the website.Which i will file separately.

**Comment by daveyarwood** [comment]
@nkonk I appreciate your effort and PR!

It ended up being pretty easy for me to deploy to Heroku, though, and set up an alda.io CNAME: http://slack.alda.io -- it's just me right now, but feel free to join and say hi! 😄 

Next steps:

- [ ] README integration (@nkonk I'll take a look at your PR and probably adapt what you have. Thanks again!)

- [ ] Update README/CONTRIBUTING mentions of Clojurians Slack to point to http://slack.alda.io instead.

- [ ] Website integration -- seems easy enough to copy-paste the script tag. I can get that knocked out.

- [ ] Inform people on Clojurians Slack #alda channel that we've moved -- I'll update the channel topic so it's easy to see. I think I'll keep that channel alive for a few months or so, just in case there are people who haven't noticed we've moved.
